bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -173,64 +180,48 @@ def mask_image(self, image):
 
 
 class PixelListColorDetector(ColorDetector):
-    def __init__(self, debug_printer, package_path, vision_config, primary_detector=False):
+    """
+    PixelListColorDetector is a ColorDetector, that is based on a lookup table of color values.
+    The color space is loaded from color-space-file at color_path (in config).
+    The color space is represented by boolean-values for RGB-color-values.
+
+    Publishes: 'field_mask' and 'dynamic_field_mask'-messages
+
+
+    The following parameters of the config dict are needed:
+        'vision_use_sim_color',
+        'field_color_detector_path_sim',
+        'field_color_detector_path'
+    """
+
+    def __init__(self, debug_printer, package_path, config):
         # type:(DebugPrinter, str, dict, bool) -> None
         """
-        PixelListColorDetector is a ColorDetector, that is based on a color space.
-        The color space is initially loaded from color-space-file at color_path (in config)
-        and optionally adjustable to changing color conditions (dynamic color space).
-        The color space is represented by boolean-values for RGB-color-values.
+        Initiating of PixelListColorDetector.
```

Comment:
Initialization of...

Reply:
Closed with 7398d9f2ebe6647e5caa15e9846bfb3e33624e37